<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1915</width>
    <height>1020</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1915</width>
    <height>1020</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1915</width>
    <height>1020</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1360</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ARI Demonstration</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="associatedStat">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>195</y>
      <width>580</width>
      <height>140</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>580</width>
      <height>140</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>580</width>
      <height>140</height>
     </size>
    </property>
    <property name="text">
     <string>Associated Categories:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="appsRatingField">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>210</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>210</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>210</width>
      <height>25</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>ex: 4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="categoriesField">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>150</y>
      <width>465</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>465</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>465</width>
      <height>25</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>ex: game, art, ...</string>
    </property>
   </widget>
   <widget class="QLabel" name="filterLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>630</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>AnjaliOldLipi</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>      - Apps Filters:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameField">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>110</y>
      <width>465</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>465</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>465</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>ex: instagram, facebook, ...</string>
    </property>
   </widget>
   <widget class="QComboBox" name="appsOrderCombo">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>230</y>
      <width>210</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>210</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>210</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="appsRatingBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>Rating</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="nameBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>App Name</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="appsOrderBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>Order by</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="statsBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>Show statistics</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="categoriesBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>Categories</string>
    </property>
   </widget>
   <widget class="QLabel" name="priceStat">
    <property name="geometry">
     <rect>
      <x>1605</x>
      <y>110</y>
      <width>285</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>285</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>285</width>
      <height>70</height>
     </size>
    </property>
    <property name="text">
     <string>Price Statistics:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1915</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1915</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1360</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>AnjaliOldLipi</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>Apps Retrieval Informations</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="statsLabel">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>70</y>
      <width>630</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>630</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>630</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>AnjaliOldLipi</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>      - Statistics:</string>
    </property>
   </widget>
   <widget class="QLabel" name="amountStat">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>110</y>
      <width>285</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>285</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>285</width>
      <height>70</height>
     </size>
    </property>
    <property name="text">
     <string>Amount by store:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="appsRunButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>580</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>580</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>Search for apps</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>167</y>
      <width>240</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="appsLtRadio">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>24</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>less than</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="appsGtRadio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>24</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>greater than</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="resultsTable">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>1815</width>
      <height>620</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1815</width>
      <height>620</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1815</width>
      <height>620</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>207</y>
      <width>240</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="appsLowHighRadio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>24</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Low to High</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="appsHighLowRadio">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>24</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>High to Low</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="filterLabel_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>630</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>AnjaliOldLipi</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>      - Reviews Filters:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="sentimentBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>110</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>Sentiment</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>795</x>
      <y>87</y>
      <width>465</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>465</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>465</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="neutralRadio">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>24</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>neutral</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="positiveRadio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>24</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>positive</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="negativeRadio">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>24</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>negative</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="reviewsOrderCombo">
    <property name="geometry">
     <rect>
      <x>795</x>
      <y>190</y>
      <width>210</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>210</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>210</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>167</y>
      <width>240</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="reviewsLowHighRadio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>24</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Low to High</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="reviewsHighLowRadio">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>24</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>High to Low</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="reviewsOrderBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>190</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>Order by</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="polarityField">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>150</y>
      <width>210</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>210</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>210</width>
      <height>25</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>ex: 4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="polarityBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>Polarity</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>795</x>
      <y>127</y>
      <width>240</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="reviewsLtRadio">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>24</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>less than</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="reviewsGtRadio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>24</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>greater than</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="reviewsRunButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>310</y>
      <width>580</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>580</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>Search for reviews</string>
    </property>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>amountStat</zorder>
   <zorder>associatedStat</zorder>
   <zorder>appsRatingField</zorder>
   <zorder>categoriesField</zorder>
   <zorder>filterLabel</zorder>
   <zorder>nameField</zorder>
   <zorder>appsOrderCombo</zorder>
   <zorder>appsRatingBox</zorder>
   <zorder>nameBox</zorder>
   <zorder>appsOrderBox</zorder>
   <zorder>statsBox</zorder>
   <zorder>categoriesBox</zorder>
   <zorder>priceStat</zorder>
   <zorder>titleLabel</zorder>
   <zorder>statsLabel</zorder>
   <zorder>appsRunButton</zorder>
   <zorder>resultsTable</zorder>
   <zorder>filterLabel_2</zorder>
   <zorder>sentimentBox</zorder>
   <zorder>reviewsOrderCombo</zorder>
   <zorder>reviewsOrderBox</zorder>
   <zorder>polarityField</zorder>
   <zorder>polarityBox</zorder>
   <zorder>groupBox_6</zorder>
   <zorder>reviewsRunButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
